packages/openactive-broker-microservice/src/models/core.d.ts

```diff
@@ -1,3 +1,30 @@
 export type OrderFeedType = 'orders' | 'order-proposals';
 export type OrderFeedIdentifier = 'OrdersFeed' | 'OrderProposalsFeed';
 export type BookingPartnerIdentifier = string;
+
+export type Opportunity = Record<string, any>;
```

Comment:
Do you want to leave a TODO here saying this type needs to be improved? Maybe even suggesting the comment you left above?

Reply:
Done